dataactcore/utils/fileD1.py

```diff
@@ -241,7 +294,7 @@ def query_data(session, agency_code, start, end, page_start, page_stop):
 
 
 def initial_query(session):
-    return session.query(
+    return session.query(*[
```

Comment:
What's the added benefit for using this notation?

Reply:
There are more than 255 columns we're querying, which essentially means more than 255 parameters for this function, which Python does not allow (throws a syntax error). This way, it's one parameter which is getting unwrapped after it's passed in.